ui/src/components/form/ScriptInput.jsx

```diff
@@ -21,6 +22,42 @@ export default function ScriptInput({ type, value, options, onValueUpdated, cols
     return <p className='error'>Input does not declare a type!</p>
   }
 
+  if (type.startsWith("options")) {
+    if (options) {
+      const optionObjects = options.map(choice => {return {value: choice, label: choice}} )
+      const menuPortal = (baseStyles, state) => ({
+        ...baseStyles,
+        zIndex: 2000 /* z-index options dropdown */
+      })
+
+      return <Select {...passedProps}
```

Comment:
We need to give this select menu a maximum width. With 6 or more selected options, the inputs column will just keep getting wider and wider, instead of acting responsively.